tools/configen/configen/config.py

```diff
@@ -10,7 +10,7 @@
 class ModuleConf:
     name: str = MISSING
     classes: List[str] = MISSING
-    _convert_: Optional[str] = None
+    default_flags: Dict[str, Any] = field(default_factory=dict)
```

Comment:
Obviously this change would have to go with some other changes.

Reply:
Yeah I'm looking into it now.

Reply:
done